op-node/p2p/signer.go

```diff
@@ -82,13 +88,14 @@ type SignerSetup interface {
 
 // LoadSignerSetup loads a configuration for a Signer to be set up later
 func LoadSignerSetup(ctx *cli.Context) (SignerSetup, error) {
-	key := ctx.GlobalString(flags.SequencerP2PKeyFlag.Name)
```

Comment:
not sure how the changes below here were made. Gotta tame my IDE formatter it looks like.